catroweb.yaml

```diff
@@ -1032,6 +1057,19 @@ components:
       items:
         $ref: '#/components/schemas/ProjectResponse'
 
+    ProjectReportResponse:
+      type: object
+      properties:
+        user_id:
+          type: string
+          example: 1234abcd-12ab-12ab-12ab-123456abcdef
+        project_id:
```

Comment:
Here the project id is already part of the path: `projects/{ID}/report`

No need for it to be sent a second time in the body